### DISCUSSION TITLE
Running tutorial examlpe of JoeFlow

### DISCUSSION THREAD

**Discussion by mohsen-fakhary** [codingjoe/joeflow#87] [Q&A]
Hi
I'm trying to run[ tutorial example of JoeFlow](https://joeflow.readthedocs.io/en/latest/tutorial/workflow.html)
When I start a workflow, it stops in second node.
In 'tasks' list in admin page, I see there's a 'has_user' task in 'scheduled' status.
How can I make the workflow to proceed?

I see no errors.
I've changed default task runner to 'celery' in conf.py

Thanks in advance

**Comment by codingjoe** [comment]
Hi @mohsen-fakhary,

Thanks for reaching out. Can you check if your Celery worker is running? Joeflow runs all machine tasks asynchronously. Therefore, you will need to have either a Celery or Dramatiq worker running to actually execute the tasks and run your workflows.
If you are new to Celery or Dramatiq, I recommend following their tutorials first. It will make your learning curve using Joeflow more pleasant.

Cheers!
Joe
